index.js

```diff
@@ -11,16 +11,18 @@ var ProductionModePlugin = require("./ProductionModePlugin");
  * - Statically extracts formatters and parsers from user code and pre-compile
  *   them into respective XXXX.
  */
-function GlobalizePlugin(attributes) {
-  this.attributes = attributes || {};
-}
+class GlobalizePlugin {
```

Comment:
On node `4.8`, I see the following sort of errors:

```
/Users/rxavier/globalizejs/globalize-webpack-plugin/index.js:14
class GlobalizePlugin {
^^^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```

Adding ["use strict" declarations](https://gist.github.com/rxaviers/ccbf280c4fa7b07b0331f7b778081362) fixes such issues, though I still see the following error when running tests (even after the added use strict). I haven't looked any further... I noticed that on Node 6, this isn't a problem.

```
  Globalize Webpack Plugin
    Production Mode
      when using default module ids
        ✓ should extract formatters and parsers from basic code
        The compiled bundle
          1) "before all" hook for "should extract formatters and parsers from basic code"
      when using named module ids
        ✓ should extract formatters and parsers from basic code
        The compiled bundle
          2) "before all" hook for "should extract formatters and parsers from basic code"
      when using hashed module ids
        ✓ should extract formatters and parsers from basic code
        The compiled bundle
          3) "before all" hook for "should extract formatters and parsers from basic code"
```